packages/main/src-old/pages/admin/AboutWorkSpace.js

```diff
@@ -8,7 +8,9 @@ import { authAxios } from "./Utils/api";
 const AboutWorkSpace = () => {
   const [orgDetails, setOrgdetails] = useState({});
   const [items, setItems] = useState(null);
-  const currentWorkspace = localStorage.getItem("currentWorkspace");
+  // const currentWorkspace = localStorage.getItem("currentWorkspace");
+
+  const currentWorkspace = localStorage.getItem("currentWorkspace") || null;
```

Comment:
Now what happens when the current workspace is `null`?
And why are you making an edit on a file in `src-old`. We're working with files in the `src` folder only

Reply:
I was mistaken, I made an edit on it before, so I had to leave it as I met it.